extension/js/common/composer.ts

```diff
@@ -1418,12 +1424,12 @@ export class Composer {
         progressCb ? (progress: number) => progressCb(progress * 0.6) : undefined);
       const message = await Mime.process(Buf.fromBase64UrlStr(raw!));
       const readableBlocks = message.blocks
-        .filter(b => b.type === 'encryptedMsg' || b.type === 'plainText' || b.type === 'plainHtml');
-      const encryptedCount = readableBlocks.filter(b => b.type === 'encryptedMsg').length;
+        .filter(b => ['encryptedMsg', 'plainText', 'plainHtml', 'signedMsg'].includes(b.type));
+      const encryptedCount = readableBlocks.filter(b => ['encryptedMsg', 'signedMsg'].includes(b.type)).length;
```

Comment:
Maybe `encryptedCount` could be renamed to `pgpBlockCount` (purpose has changed)

Reply:
Yes, agree, 
from now it stores count for both encrypted and signed blocks